frontend/src/pages/SignIn/SignInContainer.tsx

```diff
@@ -16,11 +17,16 @@ const initSignInvalue: SignInFields = {
 };
 
 const SignInContainer: React.FC = () => {
+  const goToProfile = useNavigate();
+  const auth = useGetAuth();
```

Comment:
Destructure please auth the same way as it was in example that I've send you
```suggestion
  const { mutateAsync } = useGetAuth();
```

Reply:
Fixed